.pipelines/templates/template-job-deploy-azure-env.yml

```diff
@@ -28,20 +27,9 @@ jobs:
       runOnce:
         deploy:
           steps:
-          - task: DownloadBuildArtifacts@0
-            inputs:
-              buildType: specific
-              project: ${{ parameters.vsoProjectID }}
-              pipeline: ${{ parameters.vsoConfigPipelineID }}
-              ${{ if eq(parameters.vsoConfigBuildID, 'latest') }}:
-                buildVersionToDownload: latestFromBranch
-                branchName: refs/heads/master
-              ${{ if ne(parameters.vsoConfigBuildID, 'latest') }}:
-                buildVersionToDownload: specific
-                buildId: ${{ parameters.vsoConfigBuildID }}
-              downloadType: specific
-              downloadPath: $(System.ArtifactsDirectory)/config
-            displayName: Download Config
+          - checkout: git://AzureRedHatOpenShift/RP-Config@refs/tags/${{ parameters.configVersion
+              }}
+            path: $(Agent.BuildDirectory)/s/config
```

Comment:
Better explicitly state the path it makes it easier to understand.

Reply:
Looks good. Both because it makes it explicit about the source of truth + it uses one tag for binaries and config.